### PR TITLE
Fix. Contacts update

### PR DESCRIPTION
**Pull request by Notalama**
https://docs.google.com/document/d/1VbHqoL2lwvJfprhWqgFlyRAzWjhOcNWkQo4Fff6xr2g/edit
https://docs.google.com/document/d/1lMjudb5qyuLNMrp8A7VNk54OSy6MpEyQiodn3aBr26s/edit

Task descriptions are attached in links above.

### CODE REVIEW

File: components/contacts/contact-form.js [hunk 1]
```diff
@@ -79,14 +81,12 @@ const ContactForm = () => {
       })
       .catch((err) => console.error(err));
   };
-
+  // console.log(person);
```
Comment (dmytropanontko) [inline components/contacts/contact-form.js]: remove

File: components/subscribe-for-updates/index.js [hunk 2]
```diff
@@ -7,7 +7,7 @@ export default () => {
   const [success, setSuccess] = useState('');
 
   function handleClick() {
-    const regexp = new RegExp('/^(([^<>()\\[\\]\\.,;:\\s@\\"]+(\\.[^<>()\\[\\]\\.,;:\\s@\\"]+)*)|(\\".+\\"))@(([^<>()[\\]\\.,;:\\s@\\"]+\\.)+[^<>()[\\]\\.,;:\\s@\\"]{2,})$/i');
+    const regexp = new RegExp(/[a-z0-9!#$%&'*+/=?^_`{|}~-]+(?:\.[a-z0-9!#$%&'*+/=?^_`{|}~-]+)*@(?:[a-z0-9](?:[a-z0-9-]*[a-z0-9])?\.)+[a-z0-9](?:[a-z0-9-]*[a-z0-9])?/g);
```
Comment (dmytropanontko) [inline components/subscribe-for-updates/index.js]: give a more clear name for  `regexp`

File: pages/contacts.js [hunk 3]
```diff
@@ -12,18 +13,21 @@ import SocialButton from '../components/social-buttons/main';
...
 
-const Address = ({ className }) => (
+const Address = ({ className, setSelectedCountry, selectedCountry }) => (
   <address className={className}>
     <ul className="contacts-list">
       <li
         itemProp="address"
         itemScope
         itemType="http://schema.org/PostalAddress"
+        className={(selectedCountry === 'UA' || !selectedCountry) ? 'arrow-left-triangle' : ''}
```
Comment (dmytropanontko) [inline pages/contacts.js]: use `classnames`

File: pages/contacts.js [hunk 4]
```diff
@@ -382,9 +403,22 @@ Contacts.defaultProps = {
 
 Address.propTypes = {
   className: PropTypes.string,
+  setSelectedCountry: PropTypes.func,
+  selectedCountry: PropTypes.string,
 };
 Address.defaultProps = {
   className: '',
+  setSelectedCountry: null,
```
Comment (dmytropanontko) [inline pages/contacts.js]: replace to `() => ({})`

File: pages/contacts.js [hunk 5]
```diff
@@ -382,9 +403,22 @@ Contacts.defaultProps = {
 
 Address.propTypes = {
   className: PropTypes.string,
+  setSelectedCountry: PropTypes.func,
```
Comment (dmytropanontko) [inline pages/contacts.js]: check is it required

File: pages/contacts.js (hunk 3 above)
Comment (Notalama) [reply pages/contacts.js]: Thank you, done.

File: components/subscribe-for-updates/index.js (hunk 2 above)
Comment (Notalama) [reply components/subscribe-for-updates/index.js]: Thank you, done.

**Pull request merged**